docs/2_3_common_states.adoc

```diff
@@ -98,7 +98,6 @@ Only in this case, <<fmi3DoStep>> may return with <<earlyReturn,`earlyReturn = f
 This set may be empty (<<nRequiredIntermediateVariables>> = 0) when the simulation algorithm does not intend to use intermediate update.
 Only the variables in <<requiredIntermediateVariables>> may be accessed by the simulation algorithm using <<get-and-set-variable-values,`fmi3Set{VariableType}`>> and <<get-and-set-variable-values,`fmi3Get{VariableType}`>> during <<IntermediateUpdateMode>>.
 All variables referenced in this set must be marked with the attribute <<intermediateUpdate,`intermediateUpdate = "true"`>> in <<modelDescription.xml>>.
-For Co-Simulation, these intermediate variables must be continuous-time variables.
```

Comment:
I think we should only drop the first part of the sentence until the comma - the rest should stay, right?